lib/functions_framework/cli.rb

```diff
@@ -197,6 +197,7 @@ def load_function
       raise "Undefined function: #{@target.inspect}" if function.nil?
       unless @signature_type.nil? ||
              (@signature_type == "http" && function.type == :http) ||
+             (@signature_type == "http" && function.type == :typed) ||
```

Comment:
Is it intentional that we use `--signature-type=http` for typed events? Is there a `--signature-type=typed`?

Reply:
Yep, it's intentional -- signature-type as accepted by the `gcloud` CLI needs support in the control plane / other parts of the stack. `typed` handlers are interoperable with `http` triggers and http handlers.